spec/rubocop/cop/rspec/verified_double_reference_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'verified double references' do |verified_double|
+  let(:cop_class) { RuboCop::Cop::RSpec::VerifiedDoubleReference }
+
+  context 'when EnforcedStyle is constant' do
+    let(:cop_config) do
+      { 'EnforcedStyle' => 'constant' }
+    end
+
+    it 'does not flag a violation when using a constant reference' do
+      expect_no_offenses("#{verified_double}(ClassName)")
+    end
+
+    it 'flags a violation when using a string reference' do
+      expect_offense(<<~RUBY, verified_double: verified_double)
+        %{verified_double}('ClassName')
+        _{verified_double} ^^^^^^^^^^^ Prefer `constant` references for verified doubles.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        #{verified_double}(ClassName)
+      RUBY
+    end
+
+    include_examples 'detects style',
+                     "#{verified_double}(ClassName)",
+                     'constant'
+  end
+
+  context 'when EnforcedStyle is string' do
+    let(:cop_config) do
+      { 'EnforcedStyle' => 'string' }
+    end
+
+    it 'does not flag a violation when using a string reference' do
+      expect_no_offenses("#{verified_double}('ClassName')")
+    end
+
+    it 'flags a violation when using a constant reference' do
+      expect_offense(<<~RUBY, verified_double: verified_double)
+        %{verified_double}(ClassName)
+        _{verified_double} ^^^^^^^^^ Prefer `string` references for verified doubles.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        #{verified_double}('ClassName')
+      RUBY
+    end
+
+    include_examples 'detects style',
+                     "#{verified_double}('ClassName')",
+                     'string'
+  end
+
+end
```

Comment:
CI (or `bundle exec rake`) complains about this one:
```suggestion
  end
end
```

Reply:
Sorry @t3h2mas, I force-pushed on your branch to resolve this issue. CI is green now 🎉